### DISCUSSION TITLE
麻烦给我退款

### DISCUSSION THREAD

**Discussion by liuh23** [getlantern/lantern-forum-cn#973] [Q&A]
我的回复码：78232230
要是这么慢的速度，解决不了的话，麻烦给我退款吧

**Comment by liuh23** [comment]
我的邮箱mrmaketo1@gmail.com

**Comment by MaxMKMax** [comment]
@liuh23 您好. 请您提供我支付单据. 谢谢.

**Reply by liuh23** [reply]
![Uploading 微信图片_20241021233855.jpg…]()

**Reply by liuh23** [reply]
解决好了的话，麻烦给我反馈一下

**Reply by MaxMKMax** [reply]
@liuh23 您好. 您的支付单据被GitHub屏蔽了. 您发到我的邮箱吧. 不好意思麻烦您了. 谢谢. 解决好了, 第一时间通知您.
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@liuh23 您好. 收到您的邮件了. 请问您确定要退款吗? 或者我们补偿两个月给您. 就等待我们修复好. 但之后就无法退款了.
您也确认一下是否下载了最新版本. 最新版本应该不会慢的.

**Comment by liuh23** [comment]
工作人员麻烦给我退一下款哈

**Reply by MaxMKMax** [reply]
okay

**Reply by liuh23** [reply]
你好，我这边没有收到退款呢？

**Comment by bobobun212** [comment]
@Mike777Khoo 你们修个J8 这TM都几个月了，网页都打不开，软件都没见更新过

**Reply by MaxMKMax** [reply]
@bobobun212 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by liuh23** [comment]
![image](https://github.com/user-attachments/assets/701487c9-17ac-4797-bf4e-a82583efe419)

**Comment by liuh23** [comment]
之前说给我退款的，可是我这边一直没收到退款，麻烦给我处理

**Comment by liuh23** [comment]
之前说给我退款的，可是我这边一直没收到退款，麻烦给我处理
